### ISSUE TITLE
[Shaman] Fire totems get killed from reactive shields

### ISSUE THREAD

**Issue by Aeded** [SunstriderEmu/BugTracker#2535]
**Describe the bug**
Fire totems are getting killed when they attack a mob or player that has reactive shield like Lightning shield. Probably same issue could occur with Reflective shield talent on priest

**To Reproduce**
1. Use lightning shield
2. Have other shaman put searing totem
3. Searing totem procs shield and dies from demage

**Expected behavior**
Totems should be only killable by direct attacks, they should consume the lightning shield charge but not die from it.

**Comment by blblblaz** [comment]
I agree that it doesn't make sense for Searing Totem to die like that, and I have no problem changing it. however try to find a source just for the sake of it if you can
